Hunting Queries/Corelight/CorelightRarePOST.yaml

```diff
@@ -0,0 +1,33 @@
+id: 9aab6aac-6dad-4749-b8cf-46840b6d978f
+name: Corelight - Rare PUT or POST
+description: |
+  'Query searches for rare post requests to a single location.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated